<!DOCTYPE html> 
<html>
  <head>
    <meta charset="utf-8">
    <title>isabelle berger online</title>
    <link rel="stylesheet" href="css/main.css">
  <body>
    <header>
      <h1 class="title">Isabelle Berger</h1>
      <p>***  My webpage; a work in progress! ***</p>
    </header>
    <article class="about">
      <h2> About </h2>
      <p> I am a postbaccalaureate fellow in the Oliver Lab, NIDDK, National Institutes of Health. Our lab is working to build predictive models of gene regulatory networks in <i>Drosophila melanogaster</i>. For this project I have assembled an S2 cell context-specific network model to be used as a prior, using publically available data from FlyBase, modENCODE, and the Sequence Read Archive.</p>
      <p> My background is math, but since coming to the Oliver Lab I have been doing bioinformatics work and developing skills in data munging, visualization and analysis, primarily using Python. Other skills/areas of interest include machine learning, mathematical modeling, and some limited experience with R, Matlab, and Maple. You can find most of my code and more about my projects with the Oliver Lab on my Github page (<a href="https://github.com/isabelleberger">https://github.com/isabelleberger</a>).</p>   
    </article>
    <div class="male-fly"><img src="img/male_fly.png" alt="a male drosophila"></div>
    <article class="data">
      <h2> Data </h2>
      <p> Here you can find datasets I've assembled available for download. </p>
      <section>
        <h3> Motifs </h3>
        <p> Bed file containing putative motifs for 115 transcription factors in <i>Drosophila melanogaster</i>. Motifs were mapped via FIMO using position weight matrices from the five fly motif databases on the MEME Suite <a href="http://meme-suite.org/db/motifs">webpage</a>. Results were filtered to only include motifs located in the introns or the region 1kb upstream of the gene. The bed file contains location, the FBgn of the transcription factor, and the pvalue from FIMO.</p> 
        <ul>
          <li><a href="https://hpc.nih.gov/~bergeric/tf_motifs.bed">TF motifs</a></li>
        </ul> 
      </section>
      <section>
        <h3> S2 cell ChIP-Seq Peaks </h3>
        <p> Bed file containing peak calls for 21 transcription factors in <i>Drosophila melanogaster</i>. All available S2 cell transcription factor ChIP datasets on the Sequence Read Archive were downloaded and post-processed using a standardized workflow. Peaks were called using both macs2 and spp, reflect the current FlyBase release (dm6) and have been filtered for phantom peaks (as described in Jain et al 2014). As with motifs, only peaks located in the introns or 1kb upstream were included. The bed file contains location, the FBgn of the transcription factor, and the -log 10 q value from the peak caller.</p>
        <ul>
          <li><a href="https://hpc.nih.gov/~bergeric/tf_peaks.bed">TF peaks</a></li>
        <p> Bed file containing peak calls for 8 histones in <i>Drosophila melanogaster</i>. Peaks were obtained by the same methods as transcription factors, described above. No location filtering has been done on this dataset. The bed file contains location, the name of the histone, and the -log 10 q value from the peak caller.</p>
          <li><a href="https://hpc.nih.gov/~bergeric/histone_peaks.bed">Histone peaks</a></li>
        </ul>
      </section>           
      <section>
        <h3> S2 cell prior </h3>
        <p> Three binary matrices (TF x gene) derived from public <i>Drosophila melanogaster</i> data.</p> 
        <ul>
          <li><a href="https://hpc.nih.gov/~bergeric/prior_s2/motif_matrix">Motif matrix</a></li>
          <li><a href="https://hpc.nih.gov/~bergeric/prior_s2/tf_matrix">TF ChIP matrix</a></li>
          <li><a href="https://hpc.nih.gov/~bergeric/prior_s2/histone_matrix">Histone ChIP matrix</a></li>
         </ul>
      </section>
    </article> 
    <div class="female-fly"><img src="img/female_fly.png" alt="a female drosophila"></div>  
  </body>

</html>
